variants/vueapp/src/components/FilterForm/PrioritizationPane.vue

```diff
@@ -111,49 +124,84 @@ defineExpose({ v$ })
       </div>
     </div>
 
-    <div class="col-6 pt-3" v-if="props.caddEnabled">
-      <h5 class="card-title mb-0">Pathogenicity Prioritization</h5>
+    <div class="col-xs-6 col-sm-6">
+        <div>
+        <div v-if="props.caddEnabled">
```

Comment:
please double check if this should stay or be moved to the parent (as it was before).